examples/demo/src/dashboard/NewCustomers.tsx

```diff
@@ -21,15 +22,12 @@ const NewCustomers = () => {
     const translate = useTranslate();
     const classes = useStyles();
 
-    const aMonthAgo = useMemo(() => {
-        const date = new Date();
-        date.setDate(date.getDate() - 30);
-        date.setHours(0);
-        date.setMinutes(0);
-        date.setSeconds(0);
-        date.setMilliseconds(0);
-        return date;
-    }, []);
+    const aMonthAgo = subDays(new Date(), 30);
```

Comment:
You cannot use `subMonths`? https://date-fns.org/v2.16.1/docs/subMonths

Reply:
![image](https://user-images.githubusercontent.com/99944/97843076-f5659e00-1ce8-11eb-9a7a-a0e1d8d79342.png)